plugins/wallabag.koplugin/main.lua

```diff
@@ -1019,4 +1033,14 @@ function Wallabag:onSynchronizeWallabag()
     return true
 end
 
+
+function Wallabag:onCloseDocument()
+    if self.remove_from_korader_history then
```

Comment:
Surely only if status == "complete" or status == "abandoned"?

Reply:
Ok.
I'll search where I can find 'status'

Reply:
ok got it:

status = docinfo.data.summary.status
 local docinfo = DocSettings:open(entry_path)
;)

Reply:
@Frenzie - can you help with this status . where I can find it?
It's not in `docinfo.data`  (`DocSettings:open(self.ui.document.file).data`)
for both 100% read and unfinished

Logs: 

```
04/14/20-19:59:44 DEBUG Wallabag: self.ui.document.file = /mnt/Dane/koreader/test/sample.pdf
04/14/20-19:59:44 DEBUG data is read from  /mnt/Dane/koreader/test/sample.sdr/metadata.pdf.lua
04/14/20-19:59:44 DEBUG Wallabag: DocSettings:open(self.ui.document.file)  = {
    ["filepath"] = "/mnt/Dane/koreader/test/sample.sdr/metadata.pdf.lua",
    ["data"] = {
        ["kopt_word_spacing"] = -0.2,
        ["kopt_contrast"] = 1,
        ["kopt_text_wrap"] = 0,
        ["render_mode"] = 0,
        ["highlight_drawer"] = "lighten",
        ["kopt_screen_mode"] = "portrait",
        ["gamma"] = 1,
        ["kopt_page_margin"] = 0.1,
        ["screen_mode"] = "portrait",
        ["kopt_font_fine_tune"] = 0.05,
        ["inverse_reading_order"] = false,
        ["show_overlap_enable"] = false,
        ["kopt_defect_size"] = 1,
        ["stats"] = {
            ["performance_in_pages"] = {},
            ["md5"] = "dabfd5be70e60d0d464b64ded495a3be",
            ["authors"] = "",
            ["total_time_in_sec"] = 0,
            ["notes"] = 0,
            ["title"] = "sample",
            ["highlights"] = 0,
            ["pages"] = 45,
            ["series"] = "",
            ["language"] = ""
        },
        ["doc_props"] = {
            ["creationDate"] = "D:20140516224813+08'00'",
            ["keywords"] = "",
            ["producer"] = "itext-paulo-155 (itextpdf.sf.net-lowagie.com)",
            ["authors"] = "",
            ["author"] = "",
            ["subject"] = "",
            ["title"] = "sample",
            ["description"] = "",
            ["modDate"] = "D:20140516224813+08'00'",
            ["creator"] = "pdftk 2.01 - www.pdftk.com",
            ["series"] = "",
            ["language"] = ""
        },
        ["kopt_auto_straighten"] = 0,
        ["config_panel_index"] = 1,
        ["highlights_imported"] = true,
        ["kopt_page_scroll"] = 1,
        ["kopt_detect_indent"] = 1,
        ["page_positions"] = {
            [2] = 0.69696969696969,
            [1] = 0.85353535353535
        },
        ["kopt_quality"] = 1,
        ["kopt_writing_direction"] = 0,
        ["flipping_zoom_mode"] = "page",
        ["kopt_page_gap_height"] = 8,
        ["highlight"] = {},
        ["bookmarks"] = {},
        ["readermenu_tab_index"] = 7,
        ["kopt_doc_language"] = "eng",
        ["zoom_mode"] = "pagewidth",
        ["partial_md5_checksum"] = "f40c4da7f2861191c5ec19de4d80894f",
        ["kopt_page_opt"] = 0,
        ["disable_fuzzy_search"] = false,
        ["kopt_font_size"] = 1,
        ["kopt_full_screen"] = 1,
        ["kopt_trim_page"] = 1,
        ["last_page"] = 2,
        ["rotation_mode"] = 0,
        ["kopt_max_columns"] = 2,
        ["kopt_justification"] = 3,
        ["highlight_disabled"] = false,
        ["kopt_forced_ocr"] = 0,
        ["doc_pages"] = 45,
        ["page_overlap_style"] = "dim",
        ["kopt_line_spacing"] = 1.2,
        ["normal_zoom_mode"] = "pagewidth",
        ["bbox"] = {},
        ["flipping_scroll_mode"] = false,
        ["percent_finished"] = 0.044444444444444,
        ["kopt_hw_dithering"] = 0,
        ["bookmarks_sorted"] = true
    },
    ["candidates"] = {
        [1] = {
            [1] = "/mnt/Dane/koreader/test/sample.sdr/metadata.pdf.lua",
            [2] = 1586887004
        },
        [2] = {
            [1] = "/mnt/Dane/koreader/test/sample.sdr/metadata.pdf.lua.old",
            [2] = 1586886864
        }
    },
    ["sidecar_file"] = "/mnt/Dane/koreader/test/sample.sdr/metadata.pdf.lua",
    ["history_file"] = "./history/[#mnt#Dane#koreader#test#] sample.pdf.lua",
    ["sidecar"] = "/mnt/Dane/koreader/test/sample.sdr",
    ["legacy_sidecar_file"] = "/mnt/Dane/koreader/test/sample.sdr/sample.pdf.lua"
}
```